src/components/PopupConfirm.js

```diff
@@ -0,0 +1,22 @@
+import Popup from "./Popup.js";
+
+export default class PopupConfirm extends Popup {
+  constructor(popupSelector) {
+    super({ popupSelector });
+    this._imageEditForm = this._popupElement.querySelector(".modal__form");
+  }
+
+  setSubmitAction(handler) {
```

Comment:
Great job implementing confirmation popup and the functionality related to it! In general, it can support any confirmation.